dist/LeafletEnvironmentalLayers.css

```diff
@@ -122,3 +122,71 @@
 .city-container {
 	width: 200px;
 }
+
+.popup {
+    position: absolute;
+    width: 230px;
+    height: auto;
+    background-color: white;
+    border: 1px solid black;
+    border-radius: 5px;
+    right: 250px;
+    padding: 10px;
+    font: 18px/1.5 "Helvetica Neue", Arial, Helvetica, sans-serif;
```

Comment:
Why not just make it `12px` instead of `18px/1.5`? Also can you use math expressions directly like that in CSS? Would using `calc()` be safer?